include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -16,21 +16,8 @@
 #ifndef _ONEDPL_CONFIG_H
 #define _ONEDPL_CONFIG_H
 
-#ifndef _PSTL_VERSION
-#    define _PSTL_VERSION 11000
-#    define _PSTL_VERSION_MAJOR (_PSTL_VERSION / 1000)
-#    define _PSTL_VERSION_MINOR ((_PSTL_VERSION % 1000) / 10)
-#    define _PSTL_VERSION_PATCH (_PSTL_VERSION % 10)
-#endif
-
-#define ONEDPL_VERSION_MAJOR 2022
-#define ONEDPL_VERSION_MINOR 6
-#define ONEDPL_VERSION_PATCH 0
-
-#if _ONEDPL___cplusplus >= 202002L && __has_include(<version>)
-#    include <version>
-#    define _ONEDPL_STD_FEATURE_MACROS_PRESENT 1
-#endif
+#include "../version"
```

Comment:
```suggestion
#include <oneapi/dpl/version>
```

Reply:
I think our preference is to use relative includes from within oneDPL.

Reply:
We have about 100 includes inside oneDPL code (excepting tests) like I propose.

Reply:
This is just my understanding of our convention.  I think we do both within the code.  It seems the norm in this directory is relative paths.  The norm one level up is what you suggest.

Reply:
We have about 100 includes starting from "oneapi/dpl/", about 100 includes starting from "../", and more than 200 other includes not starting from either.

Is there some real motivation why in this very case having the "full" path would be preferred?

Reply:
My assumption the full file name here is better due the header file name `version` is the same like in standard library.

Reply:
Switched to use an internal header here.